test/runtests.jl

```diff
@@ -1,4 +1,61 @@
+# This file contains code that was formerly part of Julia. License is MIT: https://julialang.org/license
+
 using AbstractFFTs
 using Base.Test
 
-# TODO
+import AbstractFFTs: plan_fft
+import Base: A_mul_B!, *
+
+mutable struct TestPlan{T} <: AbstractFFTs.Plan{T}
+    region
+    pinv::TestPlan{T}
+    TestPlan{T}(region) where {T} = new{T}(region)
+end
+
+AbstractFFTs.plan_fft(x::Vector{T}, region; kwargs...) where {T} = TestPlan{T}(region)
+
+function Base.A_mul_B!(y::Vector, p::TestPlan, x::Vector)
+    n = length(x)
+    length(y) == n || throw(DimensionMismatch())
+    c = -2im * π / n
+    @inbounds for k = 0:n-1
+        y[k+1] = dot(x, exp.(c .* k .* (0:n-1)))
+    end
+    return y
+end
+
+Base.:*(p::TestPlan, x::Vector) = A_mul_B!(x, p, copy(x))
```

Comment:
Can we fix the README too to note this?

I forgot about [this comment](https://github.com/JuliaMath/AbstractFFTs.jl/blob/bc16ae1cdbf0a169de87f9060a17bd66560c0418/src/AbstractFFTs.jl#L219-L224) in the source.